config/settings.yml

```diff
@@ -1188,6 +1188,8 @@
       :poll_method: :normal
       :restart_interval: 0.hours
       :starting_timeout: 10.minutes
+    :embedded_ansible_worker:
+      :poll: 30.seconds
```

Comment:
since the do_work is really only monitoring supervisord, I have it doing it every 30 seconds